session/session.go

```diff
@@ -442,6 +446,19 @@ func (s *session) FieldList(tableName string) ([]*ast.ResultField, error) {
 	return fields, nil
 }
 
+func (s *session) TxnInfo() *txninfo.TxnInfo {
+	txnInfo := s.txn.Info()
```

Comment:
Wait, is the `session`'s txn field possible to be changed when transaction is finished and a new transaction is started?
@cfzjywxk PTAL

Reply:
Nevermind. I thought it's a pointer field.